ISISComputingGroup/EPICS-upgrade
# Ticket 4573: Added upgrade step for checking motion set points

See https://github.com/ISISComputingGroup/IBEX/issues/4573

To test:
* Confirm that all PV name changes made in https://github.com/ISISComputingGroup/EPICS-motionSetPoints/pull/14 are converted in this script
* Confirm that if you have motion set points you are directed to upgrading them
